### PR TITLE
ParseOpData function schema changed to const

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Changed the schema params to const

### CODE REVIEW

File: tensorflow/lite/interpreter_test.cc [hunk 1]
```diff
@@ -303,7 +303,7 @@ TEST(BasicInterpreter, CheckArenaAllocation) {
 
   std::vector<int> sizes{2048, 4096, 1023, 2047, 1021,
                          2047, 1023, 2046, 0,    2048};
-  for (int i = 0; i < sizes.size(); ++i) {
+  for (size_t i = 0; i < sizes.size(); ++i) {
     interpreter.SetTensorParametersReadWrite(i, kTfLiteUInt8, "", {sizes[i]},
```
Comment (jdduke) [inline tensorflow/lite/interpreter_test.cc]: You'll need a static_cast in at least one place, otherwise revert this change.

**Review by jdduke — CHANGES_REQUESTED** [review]

File: tensorflow/lite/interpreter_test.cc (hunk 1 above)
Comment (jdduke) [reply tensorflow/lite/interpreter_test.cc]: Sorry, to be specific, SetTensorParametersReadWrite takes an integer index, not a size_t.

Comment (amitsrivastava78) [reply tensorflow/lite/interpreter_test.cc]: @jdduke , thanks for pointing that out, i have changed the code as per our suggestion i.e used static_cast. Kindly check and approve.

**Comment by amitsrivastava78** [comment]
@jdduke , thanks for pointing that out, i have changed the code as per our suggestion i.e used static_cast. Kindly check and approve.

**Review by jdduke — APPROVED** [review]
